pyproject.toml

```diff
@@ -159,8 +160,7 @@ dependencies = [
   "protobuf==3.20",  # version minimum (for tests)
   "responses",
   "tiktoken",
-  "typing-extensions==4.5.0; python_version<'3.12'",
-  "typing-extensions==4.6.0; python_version>='3.12'",
+  "typing-extensions",
```

Comment:
Unpin `typing-extensions` for testing since it pulling down the `pydantic` version.